Tests/test_file_libtiff.py

```diff
@@ -395,18 +435,44 @@ def test_blur(self):
         self.assert_image_equal(im, im2)
 
     def test_compressions(self):
+        # Test various tiff compressions and assert similar image content but reduced
+        # file sizes.
         im = hopper("RGB")
         out = self.tempfile("temp.tif")
+        im.save(out)
+        size_raw = os.path.getsize(out)
 
         for compression in ("packbits", "tiff_lzw"):
             im.save(out, compression=compression)
+            size_compressed = os.path.getsize(out)
             im2 = Image.open(out)
             self.assert_image_equal(im, im2)
 
         im.save(out, compression="jpeg")
+        size_jpeg = os.path.getsize(out)
         im2 = Image.open(out)
         self.assert_image_similar(im, im2, 30)
 
+        im.save(out, compression="jpeg", quality=30)
+        size_jpeg_30 = os.path.getsize(out)
+        im3 = Image.open(out)
+        self.assert_image_similar(im2, im3, 30)
+
+        assert size_raw > size_compressed
```

Comment:
Let's use unittest asserts for consistency:
```suggestion
        self.assertGreater(size_raw, size_compressed)
```